analysis_options.yaml

```diff
@@ -63,7 +63,6 @@ linter:
     - lines_longer_than_80_chars
     - list_remove_unrelated_type
     - literal_only_boolean_expressions
-    - missing_whitespace_between_adjacent_strings
```

Comment:
This is because of https://github.com/kevmoo/build_cli/blob/7acba4562a0d85f782922997d59556685743fe27/build_cli/lib/src/build_cli_generator.dart#L212

Reply:
You could also ignore it inline 😄